config/server.lua

```diff
@@ -136,5 +136,8 @@ return {
 
     removeSocietyMoney = function(accountName, payment)
         return exports['Renewed-Banking']:removeAccountMoney(accountName, payment)
-    end
+    end,
+
+    maxJobsPerPlayer = 1,
+    maxGangsPerPlayer = 1,
```

Comment:
Personally I'm more of a fan of using convars for these sort of things?

We haven't really discussed this at length in the developer chat, but would be worth considering

Reply:
changed to convar